dist/images/ovnkube.sh

```diff
@@ -124,7 +124,7 @@ ovn_kubernetes_namespace=${OVN_KUBERNETES_NAMESPACE:-ovn-kubernetes}
 
 # host on which ovnkube-db POD is running and this POD contains both
 # OVN NB and SB DB running in their own container. Ignore IPs in loopback range (127.0.0.0/8)
-ovn_db_host=$(getent ahostsv4 $(hostname) | grep -v "^127\." | head -1 | awk '{ print $1 }')
+ovn_db_host=${K8S_NODE_IP:-""}
```

Comment:
@Pardhakeswar Can you fix the comments in L126? It still talks about the old way of doing things.